Cellos.def

```diff
@@ -0,0 +1,14 @@
+Bootstrap: docker
+From: python:3.7-slim-bookworm
+
+%files
+    requirements.txt /requirements.txt
+
+%post
+    pip install --require-hashes --no-deps -r /requirements.txt
+
+%environment
+    export PYTHONPATH=/usr/local/bin/python
+
+%runscript
+    exec python "$@"
```

Comment:
This makes python be the command that is executed when the container is executed.
$@ should be all positional arguments.